Dashboards/CyberArk_Dashboard.json

```diff
@@ -54,7 +54,7 @@
                 },
                 {
```

Comment:
Please move the CyberArk workbook to https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks folder and follow guidance @ https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks#how-to-contribute-new-workbook to submit the workbook. 
The Dashboards folder is going to be deleted soon.